### PR TITLE
update for range annotation

### PR DESCRIPTION
**Pull request by kelseiv**
Closes https://github.com/influxdata/docs-v2/issues/2698

We'll stay tuned for when to push docs to production. @daviddkkim has kindly offered to give me the sign when we're ready!

PS One more item of note--we did hide the old video.

### CODE REVIEW

**Comment by daviddkkim** [comment]
@kelseiv  There is no such thing as "enable 1 click mode" anymore. Toggling the annotation button at the top of the page turns everything about annotations on. 

Adding a normal annotation is shift + click, adding a ranged annotation is shift + drag

File: content/influxdb/cloud/visualize-data/annotations.md [hunk 1]
```diff
@@ -21,25 +21,24 @@ Add annotations to your dashboards to provide useful, contextual information abo
...
     {{< nav-icon "dashboards" >}}
 
-2. Select an existing dashboard to add the annotation to, or [create a new dashboard](/influxdb/cloud/visualize-data/dashboards/create-dashboard/).
-3. Click **Annotations**. The **Show Annotations** option is selected by default.
-4. Select the **Enable 1-Click Annotations** option, and then in any dashboard cell, click the point in time to add the annotation.
-  {{% note %}}
-**Tip:** To move the annotation outside of the selected graph time range, first create the annotation, and then [edit the annotation](#edit-an-annotation) to update the timestamp as needed.
-  {{% /note %}}
-5. On the **Add Annotation** page:
+2. Select an existing dashboard to add the annotation to, or [create a new dashboard](/influxdb/cloud/visualize-data/dashboards/create-dashboard/). The **Annotations** option is selected by default.
```
Comment (hoorayimhelping) [inline content/influxdb/cloud/visualize-data/annotations.md:32]: @daviddkkim didn't this recently change with the annotations onboarding changes you made?

File: content/influxdb/cloud/visualize-data/annotations.md [hunk 2]
```diff
@@ -21,25 +21,24 @@ Add annotations to your dashboards to provide useful, contextual information abo
...
+3. In a dashboard cell, doing one of the following:
```
Comment (hoorayimhelping) [inline content/influxdb/cloud/visualize-data/annotations.md:33]: Is `doing` the right form here? Should it be `do`? None of the other verbs are in present participle

File: content/influxdb/cloud/visualize-data/annotations.md [hunk 3]
```diff
@@ -21,25 +21,24 @@ Add annotations to your dashboards to provide useful, contextual information abo
...
 
-   1. Verify the start time, and update if needed.
-   2. Enter a message (maximum of 255 characters) to associate with the selected start time.
-   3. Click **Save Annotation**. The annotation appears as a dotted line at the specified start time and includes details associated with the selected point.
+   - To add an annotation to a single point in time, press Shift and click the time.
```
Comment (hoorayimhelping) [inline content/influxdb/cloud/visualize-data/annotations.md:35]: should this be something more like "click the graph at the time you want to annotate" ?

File: content/influxdb/cloud/visualize-data/annotations.md (hunk 2 above)
Comment (kelseiv) [reply content/influxdb/cloud/visualize-data/annotations.md:33]:
```suggestion
3. In a dashboard cell, do one of the following:
```

File: content/influxdb/cloud/visualize-data/annotations.md (hunk 3 above)
Comment (kelseiv) [reply content/influxdb/cloud/visualize-data/annotations.md:35]:
```suggestion
   - Press Shift and click the graph at the time you want to annotate.
```

File: content/influxdb/cloud/visualize-data/annotations.md (hunk 1 above)
Comment (daviddkkim) [reply content/influxdb/cloud/visualize-data/annotations.md:32]: yes, annotations are defaulted off ! Sorry for the late reply

Comment (kelseiv) [reply content/influxdb/cloud/visualize-data/annotations.md:32]:
```suggestion
2. Select an existing dashboard to add the annotation to, or [create a new dashboard](/influxdb/cloud/visualize-data/dashboards/create-dashboard/), and then click the **Annotations** button.
```

**Review by hoorayimhelping — APPROVED** [review]

**Review by russorat — APPROVED** [review]
LGTM

**Pull request merged**